packages/app/src/cli/models/extensions/extension-specifications/checkout_ui_extension.ts

```diff
@@ -13,9 +13,10 @@ const CheckoutSchema = BaseExtensionSchema.extend({
 const spec = createExtensionSpec({
   identifier: 'checkout_ui_extension',
   externalIdentifier: 'checkout_ui',
+  externalName: 'Checkout UI',
   surface: 'checkout',
```

Comment:
Yes, extension points will have a way to specify the surface too, but since we need to support the current extension types, it makes sense to have it here too (Although it will be removed from this interface, as we should obtain the surface from the Specifiactions API)